hbase-procedure/src/main/java/org/apache/hadoop/hbase/procedure2/Procedure.java

```diff
@@ -439,6 +439,10 @@ protected void updateMetricsOnFinish(TEnvironment env, long runtime, boolean suc
       if (timeHisto != null) {
         timeHisto.update(runtime);
       }
+      Histogram timeHistoOnMasterUI = metrics.getTimeHistoOnMasterUI();
```

Comment:
You can use "instance of" and cast to OperationMetricsForMasterUI. Then no need to add the method to interface?